php-practice_1.php

```diff
@@ -1,41 +1,202 @@
 <?php
 // Q1 変数と文字列
 
+$name = "菅原沙恵";
+echo "私の名前は「" . $name . "」です。";
+
+
+
 
 // Q2 四則演算
 
+$num = 5 * 4;
+echo $num . "\n";
+echo $num / 2;
+
+
+
 
 // Q3 日付操作
 
+echo "現在時刻は、" . date("Y年m月d日 H時i分s秒") . "です。";
+
+
+
 
 // Q4 条件分岐-1 if文
 
+$device = "mac";
+
+if ($device == "windows")
+ {
+    echo "使用OSは、windowsです。"
+  ;
+  }
+else 
+  {
+    if ($device == "mac") 
+      {
+        echo "使用OSは、macです。";
+      }
+    else
+      {
+        echo "どちらでもありません。";
+      }
+    }
+
+
+
 
 // Q5 条件分岐-2 三項演算子
 
+$age = 25;
+$message = ($age < 18) ? "未成年です。" : "成人です。";
+
+echo $message;
+
+
+
 
 // Q6 配列
 
+$prefecture =
+[
+  "東京都" , "群馬県" , "茨城県" , "栃木県" ,
+  "千葉県" , "埼玉県" , "神奈川県" , "山梨県"
+];
+
+echo $prefecture[3] . "と" . $prefecture[4] . "は関東地方の都道府県です。";
+
+
+
 
 // Q7 連想配列-1
 
+$prefecture = 
+[
+  "東京都" => "新宿区" , "神奈川県" => "横浜市" , "千葉県" => "千葉市" , "埼玉県" => "さいたま市" , 
+  "栃木県" => "宇都宮市" , "群馬県" => "前橋市" , "茨城県" => "水戸市"
+]; 
```

Comment:
こちらなども1つの値ごと(カンマ1つごと)に改行しましょう！
```suggestion
$prefecture = 
[
  "東京都" => "新宿区" ,
  "神奈川県" => "横浜市" ,
  "千葉県" => "千葉市" ,
  "埼玉県" => "さいたま市" , 
  "栃木県" => "宇都宮市" ,
  "群馬県" => "前橋市" ,
  "茨城県" => "水戸市"
]; 
```